### PR TITLE
Update missing config message

### PR DESCRIPTION
**Pull request by tomchiverton**
There is no need for this message to sound so scary. letsencrypt.sh runs fine with it's defaults. I'd even consider removing this warning entirely.

### CODE REVIEW

File: letsencrypt.sh [hunk 1]
```diff
@@ -126,9 +126,7 @@ load_config() {
   IP_VERSION=
 
   if [[ -z "${CONFIG:-}" ]]; then
-    echo "#" >&2
-    echo "# !! WARNING !! No main config file found, using default config!" >&2
-    echo "#" >&2
+    echo "WARNNG: No main config file found, using defaults" >&2
```
Comment (jantore) [inline letsencrypt.sh]: Looks like a typo managed to sneak in.

**Comment by tomchiverton** [comment]
It did ?

**Comment by lukas2511** [comment]
I think this warning should exist and be big, people should at least know that there is a config, and maybe set a few important parameters so future updates don't break stuff if defaults ever change again. Making it smaller like this makes it too easy to ignore.

**Comment by tomchiverton** [comment]
Your concerns are why breaking changes (to defaults) go in a change log, or documentation, no? 
Could we at least get a --quiet option or similar? After the 30th time you see it, it's pointless as you skip over it anyway.

**Pull request closed without merging**